src/playGame.js

```diff
@@ -0,0 +1,25 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES, RULES } from "../constants/messages.js";
+import randomNumber from "../utils/randomNumber.js";
+import userInputValidation from "../utils/userInputValidation.js";
+import resultPrint from "../utils/resultPrint.js";
+import compareToNumber from "../utils/compareToNumber.js";
+
+const playGame = async () => {
+  const comNum = randomNumber();
+  while (true) {
+    const userInput = await MissionUtils.Console.readLineAsync(
+      GAME_MESSAGES.INPUT
+    );
+    await userInputValidation(userInput);
```

Comment:
```suggestion
    userInputValidation(userInput);
```
`userInputValidation` 함수는 Promise를 반환하는 비동기 함수가 아니기 때문에 `await`에는 영향이 가지 않아서 제거해줘도 좋을 것 같아요~!